src/configureStore.test.ts

```diff
@@ -70,11 +70,11 @@ describe('configureStore', () => {
 
   describe('given custom middleware', () => {
     it('calls createStore with custom middleware and without default middleware', () => {
-      const thank: redux.Middleware = _store => next => action => next(action)
-      expect(configureStore({ middleware: [thank], reducer })).toBeInstanceOf(
+      const thunk: redux.Middleware = _store => next => action => next(action)
```

Comment:
I'm not sure if this 'thank' is intentional or a type. If this is intentional I will roll back my changes.